jmxtrans/docker/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM openjdk:8u102-jdk
```

Comment:
Missing license header (note that `mvn license:format` should add those headers automatically for you)